psycop/projects/scz_bp/feature_generation/outcome_specification/add_time_from_first_visit.py

```diff
@@ -0,0 +1,26 @@
+import polars as pl
+
+from psycop.common.feature_generation.loaders.raw.load_visits import (
+    physical_visits_to_psychiatry,
+)
+
+
+def add_time_from_first_visit(df: pl.DataFrame) -> pl.DataFrame:
+    first_visit = (
+        pl.from_pandas(
+            physical_visits_to_psychiatry(
+                n_rows=None,
+                timestamps_only=False,
+                return_value_as_visit_length_days=False,
+                timestamp_for_output="start",
+            ),
+        )
+        .groupby("dw_ek_borger")
+        .agg(pl.col("timestamp").min().alias("first_visit"))
+    )
+    df = df.join(first_visit, on="dw_ek_borger", how="left")
+
+    df = df.with_columns(
```

Comment:
Bare som en neat ting, så kan du bruge pl.col("timestamp").min().over("dw_ek_borger).alias("first_visit") til at erstatte hele blokken her :-)

https://pola-rs.github.io/polars/py-polars/html/reference/expressions/api/polars.Expr.over.html

Reply:
Neat pattern! Her har jeg dog brug for aggregeringen (fra mange besøg til kun det første) - men tak for obs!